### ISSUE TITLE
Parse Error in `ntp_sources.py`

### ISSUE THREAD

**Issue by xiangce** [RedHatInsights/insights-core#727]
Another error found in ntp_sources.py when test with some insights-cli:

~~~
Reducer telemetry.rules.plugins.kernel.ntpd_not_reset_leap_status.report failed                
Traceback (most recent call last):                                                                                                                                                             
  File "/home/liuxc/work/insights/local_insights_CLI/insights-core/insights/core/reducer.py", line 158, in run_reducer                                                                         
    r = plugins.DELEGATES[func](local, shared)                                                                                                                                                 
  File "/home/liuxc/work/insights/local_insights_CLI/insights-core/insights/core/plugins.py", line 483, in __f                                                                                 
    return executor(func, local, broker, shared=is_shared, cluster=cluster, emitter=emitter)   
  File "/home/liuxc/work/insights/local_insights_CLI/insights-core/insights/core/plugins.py", line 431, in default_executor                                                                    
    return func(local, broker)                                                                 
  File "/home/liuxc/work/insights/local_insights_CLI/insights-plugins/telemetry/rules/plugins/kernel/ntpd_not_reset_leap_status.py", line 46, in report                                        
    leap_val = shared[NtpqLeap].leap                                                                                                                                                           
  File "/home/liuxc/work/insights/local_insights_CLI/insights-core/insights/parsers/ntp_sources.py", line 84, in leap                                                                          
    if self.data:                              
AttributeError: 'NtpqLeap' object has no attribute 'data' 

~~~

**Comment by xiangce** [comment]
@koalakangaroo - Since I can reproduce this issue in my env currently, I will try to fix it at first.

**Comment by csams** [comment]
Fixed in #728 and #729.
